anchor/anchors.go

```diff
@@ -76,3 +76,22 @@ func (a Anchor) SubType() string {
 func (a Anchor) Value() []string {
 	return a.value
 }
+
+// GetSources returns the anchors which identify how and when an attribute value was acquired.
+func GetSources(anchors []*Anchor) (sources []*Anchor) {
+	return filterAnchors(anchors, AnchorTypeSource)
+}
+
+// GetVerifiers returns the anchors which identify how and when an attribute value was verified by another provider.
+func GetVerifiers(anchors []*Anchor) (sources []*Anchor) {
+	return filterAnchors(anchors, AnchorTypeVerifier)
+}
+
+func filterAnchors(anchors []*Anchor, anchorType Type) (result []*Anchor) {
+	for _, v := range anchors {
+		if v.Type == anchorType {
```

Comment:
Does the Anchor Object include the AnchorType?
If so what's the value of the type e.g 'source' or the old e.g '1.3.6.1.4.1.47127.1.1.1'?

Reply:
Discussed in https://lampkicking.atlassian.net/browse/SDK-683: having enum type is fine for the foreseeable future, and we won't expose OID